UI/lib/ui-header.html

```diff
@@ -16,26 +16,25 @@
 <head>
     <title>[% title %]</title>
     <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
-    [%+# HTML Snippet, for import only %]
-    [%+#
+    [%-#
         # source comment only!
         #
         # don't specify a title on the stylesheets: we want them to be
         # *persistent*
         # http://www.w3.org/TR/html401/present/styles.html#h-14.3.1
-    %]
+    -%]
```

Comment:
Is there a reason you removed all the minus signs? I mean, they are there to strip the leading spaces, in order to result in a "nice" HTML output.

Reply:
I added them for that exact purpose.

Reply:
Oops...